packages/mdc-notched-outline/foundation.js

```diff
@@ -99,7 +99,12 @@ class MDCNotchedOutlineFoundation extends MDCFoundation {
     const height = this.adapter_.getHeight();
     const cornerWidth = radius + 1.2;
     const leadingStrokeLength = Math.abs(12 - cornerWidth);
-    const paddedNotchWidth = notchWidth + 8;
+
+    // If the notchWidth is 0, the label doesn't need padding.
```

Comment:
This comment sounds a bit weird... if the notchWidth is 0, there's no label, right?